crates/ruff_linter/src/logging.rs

```diff
@@ -34,6 +35,26 @@ macro_rules! warn_user_once_by_id {
     };
 }
 
+pub static MESSAGES: Lazy<Mutex<FxHashSet<String>>> = Lazy::new(Mutex::default);
```

Comment:
A small nit for clarity.

```suggestion
pub static PREVIOUS_MESSAGES: Lazy<Mutex<FxHashSet<String>>> = Lazy::new(Mutex::default);
```